dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/task/BaseTaskProcessor.java

```diff
@@ -190,9 +190,9 @@ public void init(@NonNull TaskInstance taskInstance, @NonNull ProcessInstance pr
 
     @Override
     public boolean action(TaskAction taskAction) {
-        String threadName = Thread.currentThread().getName();
+        String oldTaskInstanceLogPathMdc = Optional.ofNullable(LogUtils.getTaskInstanceLogFullPathMdc()).orElse(null);
```

Comment:
This looks identical to 

```suggestion
        String oldTaskInstanceLogPathMdc = LogUtils.getTaskInstanceLogFullPathMdc();
```